docs/source/open-software-week/big-imaging-data.md

```diff
@@ -1,2 +1,58 @@
+* engage in hands-on tutorials to handle big microscopy data with open-source software
+* build connections between the imaging and software engineering community
+
+The Big Imaging Data track will include
+* hands-on introductory tutorials to open-source libraries `zarr` and `dask`
+* participant-led discussions of community, collaboration and careers in Big Imaging Data
+
+::: {admonition} Target audience
+:class: note
+
+This course is designed for imaging facility staff, image analysts, and imaging researchers interested in getting hands-on experience in, and building community around, free open-source tools to process big imaging data.
+:::
+
+## Course overview
+
+__Tuesday morning:__
+The technical side of the track will start with an introduction to chunked file formats for big imaging data, such as `OME-zarr`.
+We will then proceed to work on hands-on tutorials to
+- read and write big imaging data using `zarr`
+- process big imaging data lazily and in-parallel with `dask`
+
+__Tuesday afternoon:__
+The community side of the track will start with a very short introduction to existing community efforts in both software engineering [^1]_, imaging[^2]_ and image analysis[^3]_.
+The participants will then join small discussion groups of their choice circling around the wider question: Where next for careers and community in Big Imaging Data?
+The discussions will then be fed back to the wider group, and result in the publication of a blog after the course.
+
+## Instructors
+- [Alessandro Felder](https://github.com/alessandrofelder)
+- [Igor Tatarnikov](https://github.com/IgorTatarnikov)
+- [Adam Tyson](https://github.com/adamltyson)
+
+## Pre-requisites
+
+### Hardware
+As this is a hands-on workshop, you will need to bring your own laptop. Any fairly recent laptop will be suitable, you don't need a GPU etc.
+
+### Software
+Please ensure you have the following installed:
+- A Python IDE such as:
+  - [Visual Studio Code](https://code.visualstudio.com/) with the [Python extension](https://marketplace.visualstudio.com/items?itemName=ms-python.python)
+  - [PyCharm](https://www.jetbrains.com/pycharm/)
+- A working `conda` (or `mamba`) installation. If you don't have it, install via [Miniforge](https://github.com/conda-forge/miniforge).
+- A working [Git](https://git-scm.com/) installation.
+
+We will email you at least a week before the event with instructions on installing any additional required software.
+
+### Python knowledge
+If you're new to Python, we recommend attending our __Intro to Python__ workshop on Monday, or completing an equivalent course beforehand.
+This hands-on session will cover the basics, including data types, control flow, functions, and core libraries—a great way to get up to speed before this event.
+
```

Comment:
Is it worth specifically promoting the other parts of OSW that might be related, e.g. brainglobe?